src/pages/guides/reference/configuringproxy.md

```diff
@@ -0,0 +1,15 @@
+# Configuring Proxy 
+
+Runtime does not expose egress IPs due to security reasons. If customers need a way to securely communicate with their backend services, they can use a proxy in between their system and Runtime. 
+
+This can be done as following: 
```

Comment:
I feel like it should be "can be done using one of the following solutions" and not "done as following" since that and the use of numbered list makes it sound like you should have to do all of the items, not necessarily just one